### PR TITLE
Next speaker button only allows current user to hit next

### LINKED ISSUE #75: Prevent non-speakers from advancing the queue

### PR DESCRIPTION
**Pull request by DanLindeman**
Closes #75

### CODE REVIEW

File: tests/test_api_rooms.py [hunk 1]
```diff
@@ -178,15 +178,11 @@ def describe_detail_next():
     def describe_POST():
 
         def it_returns_200_and_updated_queue(client, populated_room):
-            status, content = load(client.post("/api/rooms/foobar/next"))
+            status, content = load(client.post("/api/rooms/foobar/next", "John Doe"))
```
Comment (jacebrowning) [inline tests/test_api_rooms.py]: I think you want to use `data=params` like in the other tests in this file.

**Comment by DanLindeman** [comment]
This still needs an affirmative test, where the current speaker is allowed to remove their current card from the queue

File: tests/test_api_rooms.py [hunk 2]
```diff
@@ -177,9 +177,36 @@ def describe_detail_next():
 
     def describe_POST():
 
-        def it_returns_200_and_updated_queue(client, populated_room):
-            status, content = load(client.post("/api/rooms/foobar/next"))
+        def it_returns_200_does_not_update_queue(client, populated_room):
```
Comment (jacebrowning) [inline tests/test_api_rooms.py]: What does this test do? The name is unclear to me.

Comment (DanLindeman) [inline tests/test_api_rooms.py]: This is supposed to test the negative case which ignores the non-current speaker. I have moved it down and renamed the test.

**Pull request merged**